.github/composite/README.md

```diff
@@ -0,0 +1,38 @@
+# Composite Github Actions
```

Comment:
I would change the folder hierarchy as following:
```
.github/
    actions/
        gradle/
            exp1/
            exp2/
            exp3/
            download/
        maven/
            exp1/
            exp2/
            exp3/
            download/
```

Reply:
done, please note that Maven is not yet in the scope of this PR